tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -79,7 +79,8 @@ mapping:
       - Kata Containers CI / kata-containers-ci-on-push / run-basic-amd64-tests / run-nydus (lts, dragonball)
       - Kata Containers CI / kata-containers-ci-on-push / run-basic-amd64-tests / run-nydus (lts, qemu)
       - Kata Containers CI / kata-containers-ci-on-push / run-basic-amd64-tests / run-nydus (lts, stratovirt)
-      - Kata Containers CI / kata-containers-ci-on-push / run-cri-containerd-tests-s390x / run-cri-containerd (active, qemu)
+      - Kata Containers CI / kata-containers-ci-on-push / run-basic-s390x-tests / run-cri-containerd (active, qemu)
+      - Kata Containers CI / kata-containers-ci-on-push / run-kata-coco-tests / run-k8s-tests-coco-nontee (qemu-coco-dev, nydus, guest-pull)
```

Comment:
Oh dear, maybe I should have reviewed this PR before trusting it was ready to go in. @BbolroC - what is the reason for re-requiring the non-tee tests in this PR? The KBS problem hasn't been fixed?

Reply:
@stevenhorsman that was done by @fidencio 😉

Reply:
@stevenhorsman @fidencio here is a PR to disable it. https://github.com/kata-containers/kata-containers/pull/11201

Reply:
Yeah, my bad, that was a screw up from my side when doing the force push to resolve merge conflicts.
However, when I did that the run-kata-coco-tests was not marked as non-required yet.

Reply:
It sounds like it's just my fault for not checking it was up-to-date before merging, so I hereby hand in my non-non-talented engineer status 😭